src/DataShare/DataShare/DataSetMapping/NewAzDataShareDataSetMapping.cs

```diff
@@ -104,7 +104,7 @@ public class NewAzDataShareDataSetMapping : AzureDataShareCmdletBase
         [Parameter(
             Mandatory = true,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Azure Storage Account ResourceId",
+            HelpMessage = "Azure Storage Account ResourceId that is the source of the data being shared.",
```

Comment:
Help message for AccountName should be updated as well. Also, please check Get/Update/Remove/SetAzDataShareDataSetMapping.

Reply:
done